src/com/epam/javacourse/homework4/GreenhouseManagement.java

```diff
@@ -1,14 +1,20 @@
 package com.epam.javacourse.homework4;
 
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Scanner;
 
 public class GreenhouseManagement implements Greenhouse {
+    private static final Path FILE_NAME = Paths.get("C:\\Users\\Sviatlana_Shedava\\IdeaProjects\\SviatlanaShedava5\\src\\com\\epam\\javacourse\\homework4\\plants.txt");
+    private static final String DELIMITER = ";";
     private final List<Plant> reservedPlants;
     private final List<Plant> availablePlants;
     public double temperature;
 
-    public GreenhouseManagement() {
+    public GreenhouseManagement(Path path) {
```

Comment:
Why is this parameter never used?
It's a good idea to pass the path using the constructor, let's go with is. 
And consider getting rid of `FILE_NAME` in this case.